lib/tree_stand/node.rb

```diff
@@ -11,16 +11,49 @@ class Node
     extend Forwardable
     include Enumerable
 
+    # @!method changed?
+    #   @return [Boolean] true if a syntax node has been edited.
+    # @!method child_count
+    #   @return [Integer] the number of children nodes.
+    # @!method extra?
+    #   @return [Boolean] true if the node is *extra* (e.g. comments).
+    # @!method has_error?
+    #   @return [Boolean] true if the node is a syntax error or contains any syntax errors.
```

Comment:
What does "contain" mean here?  Immediate children, or any descendant?

Reply:
Doc and source code of tree-sitter is not clear. I will try to provide an answer when I fiddle with it.

Reply:
According to what I saw in my own work, it's any descendant.